docs/types.rst

```diff
@@ -218,18 +218,6 @@ Members:
     It is possible to use an array of bytes as ``byte[]``, but it is wasting a lot of space, 31 bytes every element,
     to be exact, when passing in calls. It is better to use ``bytes``.
 
-Dynamically-sized byte array
```

Comment:
Perhaps it makes sense to at least put a stub here in case people search for such types?

So basically only the paragraph at the end of this section has to be moved.

What do you think, @axic?